e2e/smoke-tests-suite/panels_smokescreen.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { e2e } from '@grafana/e2e';
+
+// TODO: Better way to generate this list (ideally including all panels >=beta state) (list is case sensitive)
```

Comment:
In terms of mitigating test flakiness / maintaining this test, figuring out a way to correctly generate this list is pretty important